diff -urN asciijump-1.0.2beta.orig/asciijump.desktop asciijump-1.0.2beta/asciijump.desktop
--- asciijump-1.0.2beta.orig/asciijump.desktop	2004-11-30 13:21:40.754047784 +0100
+++ asciijump-1.0.2beta/asciijump.desktop	2004-11-30 13:21:56.635633416 +0100
@@ -6,6 +6,6 @@
 Exec=asciijump
 Terminal=true
 Type=Application
-Categories=GNOME;Application;Game;BoardGame;
+Categories=Game;SportsGame;
 StartupNotify=true
 Encoding=UTF-8
